template/README.template.md

```diff
@@ -0,0 +1,22 @@
+# Server Folder
+
+This folder contains the server code for the project.
+It contains the following files:
+
+- `bin/server.dart` - The main entry point for the server.
+- `bin/middlewares.dart` - Contains all the middlewares for the server. Contains all CSP, CORS, and other security headers.
+- `Dockerfile` - The Dockerfile for the server to build it.
```

Comment:
Maybe `The Dockerfile to build a docker image containing the server` is more precise 🤔

Reply:
Done